tests/python/test_aot.py

```diff
@@ -202,7 +202,7 @@ def init(d: ti.i32, density1: ti.any_arr(), density2: ti.any_arr(),
 
 @ti.test(arch=ti.opengl)
 def test_opengl_exceed_max_ssbo():
-    # 7 ndarrays + gtmp + args > 8 (maximum allowed)
+    # 8 ndarrays + args > 8 (maximum allowed)
```

Comment:
Do we really want this test? This seems a bit arch specific & a lot of devices support more

Reply:
Yea this test was mainly a self reminder to double check how many ssbos we create in normal cases - we can remove it later :D